cli.js

```diff
@@ -143,20 +154,38 @@ sade2
     'File extensions allowed in the bundle.  (default js,cjs,mjs,ts,tsx)'
   )
   .option('--config', 'Path to the config file')
-  .action((input, opts) => {
+  .action(async (input, opts) => {
     let config
     try {
       if (opts.config) {
-        config = lilconfigSync('playwright-test').load(
-          path.resolve(opts.config)
-        )
+        config = await lilconfig('playwright-test', {
+          loaders: {
+            '.js': loadEsm,
+            '.mjs': loadEsm,
+          },
+        }).load(path.resolve(opts.config))
```

Comment:
can hoist this up to avoid duplication please
```js
{
  loaders: {
    '.js': loadEsm,
    '.mjs': loadEsm,
  },
}
```

Reply:
Done!